src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -106,8 +60,8 @@ const TopicPage = ({ pageData }) => {
           <AdContainer slotType="leaderboard" />
         </>
       )}
-      <OuterWrapper role="main">
-        <InnerWrapper>
+      <div css={styles.main} role="main">
```

Comment:
I think this can be an HTML `<main>` element

Reply:
done